src/main/java/uk/gov/hmcts/divorce/common/notification/FinancialOrderRequestedNotification.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.divorce.common.notification;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.common.config.EmailTemplatesConfig;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.LanguagePreference;
+import uk.gov.hmcts.divorce.notification.ApplicantNotification;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import java.util.Map;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.FINANCIAL_ORDER_REQUESTED_NOTIFICATION;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.RESPONDENT_FINANCIAL_ORDER_REQUESTED_NOTIFICATION;
+
+@Component
+@Slf4j
+public class FinancialOrderRequestedNotification implements ApplicantNotification {
+
+    @Autowired
+    private NotificationService notificationService;
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Autowired
+    private EmailTemplatesConfig config;
+
+    @Override
+    public void sendToApplicant1(final CaseData caseData, final Long caseId) {
+
+        final String email = caseData.getApplicant1().getEmail();
+        final LanguagePreference languagePreference = caseData.getApplicant1().getLanguagePreference();
+
+        log.info("Sending financial order requested notification to applicant 1 for case : {}", caseId);
+
+        final Map<String, String> templateVars =
```

Comment:
There's a fee in the email template where it says "Applying for a contested financial order cost **£275**". We might be able to pass that in as a variable from the payment service to avoid the updating template when the [fees increase](https://hmcts-reform.slack.com/archives/C01G1KRSEN4/p1709728315133079) 🤔

Reply:
Yeah thats a good suggestion
I think the same thing was mentioned about Divorce fee and any future changes.
Should we make this change along with other such changes that might be needed?

Reply:
@adamg-hmcts I have implemented the functionality to get the FinancialOrderOnNotice fee from the Payment Service.